docker/nginx.conf

```diff
@@ -0,0 +1,49 @@
+user  nginx;
+
+# XXX-Anand: Oct 2013
+# Increased the worker_processes to allow more workers to share the load of https.
+
+#worker_processes  1;
+worker_processes 4;
+
+
+error_log  /var/log/nginx/error.log;
+pid        /var/run/nginx.pid;
+
+events {
+    worker_connections  1024;
+    # multi_accept on;
+}
+
+http {
+    ssl_protocols  TLSv1 TLSv1.1 TLSv1.2;
+    ssl_session_cache   shared:SSL:50m;
+    ssl_session_timeout 10m;
+
+    include       /etc/nginx/mime.types;
+
+    server_names_hash_bucket_size   64;
+    types_hash_bucket_size 64;
+
+    log_format iacombined '$seed$remote_addr $host $remote_user [$time_local] "$request" $status $body_bytes_sent "$http_referer" "$http_user_agent" $request_time';
```

Comment:
Are all of these host variables getting passed through correctly, especially when we're bouncing between the external facing nginx and one of the cover services (which I imagine is actually fine)